virtual_rainforest/models/hydrology/model_schema.json

```diff
@@ -11,6 +11,12 @@
                     "exclusiveMinimum": 0,
                     "default": 0.5
                 },
+                "init_groundwater_saturation": {
```

Comment:
I'd spell this out fully as `initial_groundwater_saturation` because you already have a property starting with `initial_` and it's good to be consistent.